corehq/apps/hqwebapp/forms.py

```diff
@@ -21,6 +24,23 @@ def clean_username(self):
         username = self.cleaned_data['username'].lower()
         return username
 
+    def clean(self):
+        lockout_message = mark_safe(_('Sorry - you have attempted to login with an incorrect password too many times. Please <a href="/accounts/password_reset_email/">click here</a> to reset your password.'))
```

Comment:
In order to lock out out a user, we need to link login attempts to the username, so the lockout itself would let the attacker know they have a valid username. Given that, I am not really sure it matters, and this lets users who have to go through a password reset process know the right email to initiate the reset with.